src/templates/article.js

```diff
@@ -145,6 +145,7 @@ const Article = props => {
     const childNodes = dlv(__refDocMapping, 'ast.children', []);
     const contentNodes = getContent(childNodes);
     const meta = dlv(__refDocMapping, 'query_fields');
+    const og = meta.og || {};
```

Comment:
We may need to create an SEO component (there is a gatsby doc on that) so we can consolidate all the `og` `twitter` `canonical` stuff

Reply:
I'm assuming this can be done separately?

Reply:
If all the varying requirements can be met in this PR without bringing them altogether, sure :)

Reply:
Actually, looking at this, I think it would be better to move the twitter and og and canonical stuff into a component now so nothing is over-ridden. You'll also have to remove the twitter directive from the things we pass to the component factory.

Reply:
Yeah I think now is the time to bring them all together.